wireless/bluetooth/bt_services.c

```diff
@@ -0,0 +1,241 @@
+/****************************************************************************
+ * wireless/bluetooth/bt_services.c
+ *
+ *   Copyright (c) 2016, Intel Corporation
+ *   All rights reserved.
```

Comment:
Is it possible to contribute this under https://www.apache.org/licenses/LICENSE-2.0 ?
Also, is this file really created in 2016?

Reply:
I have no legal training so I keep the license of the code I got inspiration from.

Reply:
@btashton @jerpelea what is your opinion?

Reply:
I'm completely fine to keep code as is, but:
1. We should be Intel copyright if code is originally produced by Intel. If you are currently working at Intel and contribute this code on behalf of Intel, that is ok.
2. The year of creation. If you contribute a brand new file with 2016 that is fine to accept if the original file on your PC is from 2016.

I'm also not a legal expert, so asking the opinion of people that spent more time on learning legal issues.
We have a big heritage from the time when the project was moved to Apache, so I want to make sure that the project is under control of all new non-Apache licensed files. 

Please ignore my comment if that is not relevant. I didn't intend to limit anyhow volunteers contributions. Thank you for pointing out other places. I will take a look.

Reply:
@ldube thank you for your effort. I'm also volunteering to work on this project.

Reply:
Thank you for the explanation! I was confused by the term "inspired" and thought that you wrote the code by yourself. Of course the copied parts should be provided with a proper copyright notice.

Reply:
@ldube could you please elaborate more about it:
"I would suggest that you delete any inaccurate comments instead of asking me to ignore them. It is hard to build trust online and you are not helping."

I think Petro is completely fine and honest on his suggestions